documentation/docs/steps/abapLandscapePortalUpdateAddOnProduct.md

```diff
@@ -0,0 +1,55 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+- Please make sure, that the service landscape-portal-api-for-s4hc with plan api was assigned as entitlement to the subaccount, where you are about to deploy addon product.
```

Comment:
As far as I know from the last discussions, this step will only work for Embedded Steampunk. If yes, you could mention this in the step documentation